lib/components/molecules/Toast/Toast.style.js

```diff
@@ -0,0 +1,48 @@
+import { css } from 'styled-components';
+
+const ToastStyles = css`
+  z-index: 1;
+  border: none;
+  display: flex;
+  align-items: center;
+  border-radius: 0;
+  position: relative;
+  min-height: 80px;
+  min-width: 400px;
+  top: 30px;
+  right: 30px;
+  background-image: url('.../static/images/warning.png');
+  ${props => `
+    ${props.theme.colors.CTA_PRIMARY};
+    ${props.theme.typography.FONT_DEFAULT};
+    padding:0 ${props.theme.spacing.unit2};
+  `};
+  ${props =>
+    props.variation === `info`
+      ? `
+    ${props.theme.colors.CTA_SECONDARY};
+  `
+      : ``};
+  ${props => (props.inheritedStyles ? props.inheritedStyles : '')};
+  position: fixed;
+  .toast_text {
+    display: flex;
```

Comment:
Don't understand the purpose of 

display: flex;
    align-items: center;
    justify-content: space-between;
    width: 250px;

Reply:
Fixed